test/tests/viewer.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+/* global Viewer, describe, it, chai, afterEach */
+var expect = chai.expect;
```

Comment:
This needs to run in phantomjs which is essentially an older version of chrome. It won't run with any es6.